cpp/src/ray/address_helper.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <dlfcn.h>
+#include <stdint.h>
+
+namespace ray {
+namespace api {
+
+/// A base address which is used to calculate function offset
+extern uintptr_t dylib_base_addr;
```

Comment:
This is the base address of what? the current library?

Reply:
Yes, current library.